cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControl.java

```diff
@@ -495,22 +495,22 @@ public Secret generateApiSecret(PasswordGenerator passwordGenerator, Secret oldS
 
     /**
      * Generate the Secret containing the Cruise Control certificate signed by the cluster CA certificate used for TLS based
-     * internal communication with Kafka
-     * It also contains the related Cruise Control private key.
+     * internal communication with Kafka. It also contains the related Cruise Control private key.
      *
-     * @param namespace Namespace in which the Cruise Control cluster runs
-     * @param kafkaName Name of the Kafka cluster (it is used for the SANs in the certificate)
-     * @param clusterCa The cluster CA.
-     * @param isMaintenanceTimeWindowsSatisfied Indicates whether we are in the maintenance window or not.
-     *                                          This is used for certificate renewals
+     * @param namespace                             Namespace in which the Cruise Control cluster runs
+     * @param clusterName                           Name of the Kafka cluster (it is used for the SANs in the certificate)
+     * @param clusterCa                             The cluster CA.
+     * @param existingSecret                        The existing secret with Kafka certificates
+     * @param isMaintenanceTimeWindowsSatisfied     Indicates whether we are in the maintenance window or not.
+     *                                              This is used for certificate renewals
      *
      * @return The generated Secret.
      */
-    public Secret generateCertificatesSecret(String namespace, String kafkaName, ClusterCa clusterCa, boolean isMaintenanceTimeWindowsSatisfied) {
+    public Secret generateCertificatesSecret(String namespace, String clusterName, ClusterCa clusterCa, Secret existingSecret, boolean isMaintenanceTimeWindowsSatisfied) {
```

Comment:
I see the method doing the same stuff in other classes is just called `generateSecret` (i.e. EntityTopicOperator) while `generateCertificatesSecret` in others (i.e. here and KafkaCluster). Would you mind having a consistent name? Wdyt? I would be to have `generateCertificatesSecret` on all.

Reply:
Yeah, that makes sense. Done.